_posts/2021-06-16-android-flexible-configuration-with-deferred-resources.md

```diff
@@ -0,0 +1,238 @@
+---
+title: "Android: Flexible configuration with Deferred Resources"
+excerpt: Deferred Resources provides flexibility beyond the standard resource-resolution approaches.
+tags: Mobile Android Open-source Deferred-Resources
+authors:
+- Drew Hamilton
+header:
+  teaser: /assets/images/post/05_color_configuration.png
+  teaser_alt: Deferred Resources
+category: Mobile
+---
+
+Our feature libraries, which include UI, have a number of varying requirements regarding
+configuration by our customers. (My colleague Hari wrote about one such library's requirements
+[here]({% post_url 2020-08-14-android-configuration-driven-ui-from-epoxy-to-compose %}).) One
+category of common configurations is resources—text, colors, images—which may be defined in various
```

Comment:
Resources (text, colors, images) are a common type of configuration, and can be defined in various ways, typically code, resources or theme attributes.

Reply:
Used a paraphrase of your suggestion, let me know what you think.